ui/src/components/LandscapeWayfinding.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import * as Popover from '@radix-ui/react-popover';
+import { setCalmSetting, SettingsState } from '../state/settings';
+import { Dialog, DialogContent } from './Dialog';
+import { Button } from './Button';
+import { useCharges } from '../state/docket';
+import { GroupLink } from './GroupLink';
+
+interface Group {
+  title: string;
+  description: string;
+  icon: string;
+  color: string;
+  link: string;
+}
+
+interface App {
+  title: string;
+  description: string;
+  image: string;
+  color: string;
+  link: string;
+}
+
+const groups: Record<string, Group> = {
+  foundation: {
+    title: 'Urbit Foundation',
+    description: 'Learn about the Urbit Project',
+    icon: 'https://interstellar.nyc3.digitaloceanspaces.com/battus-datsun/2022.11.07..19.39.22-Sig.png',
+    color: 'bg-black',
+    link: '/apps/groups/groups/~halbex-palheb/uf-public',
+  },
+  door: {
+    title: 'door.link',
+    description: 'A cult of music lovers',
+    icon: 'https://www.door.link/logowhite.svg',
+    color: 'bg-black',
+    link: '/apps/groups/groups/~natnex-ronret/door-link',
+  },
+  tlonPublic: {
+    title: 'Tlon Public',
+    description: 'A place to ask for help',
+    icon: 'https://sfo3.digitaloceanspaces.com/zurbit-images/dovsem-bornyl/2022.6.16..19.11.20-flooring.jpeg',
+    color: 'bg-yellow-500',
+    link: '/apps/groups/groups/~nibset-napwyn/tlon',
+  },
+};
```

Comment:
would be nice to maybe just use the light groups scry for this so we don't have to hardcode it